images/win/scripts/Installers/Install-Stack.ps1

```diff
@@ -0,0 +1,19 @@
+################################################################################
+##  File:  Install-Stack.ps1
+##  Desc:  Install Stack for Windows
+################################################################################
+
+Write-Host "Get the latest Stack version..."
+$StackReleasesJson = Invoke-RestMethod "https://api.github.com/repos/commercialhaskell/stack/releases/latest"
+$DownloadUrl = $StackReleasesJson.assets | Where-Object { $_.name.EndsWith("windows-x86_64.zip") } | Select-Object -ExpandProperty "browser_download_url" -First 1
```

Comment:
I think `"windows-x86_64.zip"` should be moved to the variable `$downloadFilePattern`